docs/modules/ROOT/pages/knowledge.adoc

```diff
@@ -0,0 +1,14 @@
+Can I restrict a function to EOAs only?
```

Comment:
The answer to this question is closer to "no", but the content here seems to be saying "yes".

The key part that is missing is that although `isContract` (equivalently, `addr.code.length > 0`) may _seem_ to differentiate contracts from EOAs, it can only say that an address is currently a contract, and its negation *(that an address is not currently a contract) does not imply that the address is an EOA.

Reply:
I see, please check the new version to see if its clear enough after the update.